modules/creating-a-machine-pool-cli.adoc

```diff
@@ -4,14 +4,14 @@
 
 :_content-type: PROCEDURE
 [id="creating_machine_pools_cli_{context}"]
-= Creating a machine pool using the ROSA CLI
+= Creating a machine pool using the {product-title} (ROSA) CLI (`rosa`)
```

Comment:
In the interests of keeping titles readable, I think we should just use "ROSA CLI" here - this ends up being ~15 words long when rendered otherwise

Reply:
Good catch. I believe, we should revoke the update here and use "ROSA CLI". I found most of instances of "the ROSA CLI" in title for ROSA docs.  
Example: https://docs.openshift.com/rosa/rosa_cli/rosa-get-started-cli.html
Please let me know if you have other thoughts. Thank you!